packages/lodestar/src/cli/commands/validator.ts

```diff
@@ -2,13 +2,11 @@
  * @module cli/commands
```

Comment:
Maybe for another PR, but we should totally have the CLI in a different package and have that require both the beacon chain and the validator.

Reply:
Yeah but here's a problem, debugging on the fly could be a pain. You'd have to keep lerna linking no?

Reply:
CLI in a separate package would be nice to separate concerns. Debugging is always a little trickier with more packages, but the monorepo makes debugging easier.
Lerna should handle the linking, just rebuild the needed packages.